.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG VARIANT=8.0
+FROM mcr.microsoft.com/vscode/devcontainers/dotnet:${VARIANT}
+ARG NODE_VERSION="none"
```

Comment:
I don't think PlanthorIdentity needs a node for any reason. (at least for now it is using bootstrap 4)